packages/extension-chakra-storefront/src/pages/home/index.jsx

```diff
@@ -83,6 +83,7 @@ const Home = () => {
         dataCloud.sendViewPage(pathname)
     }, [])
 
+    return <div>Home</div>
```

Comment:
TODO: will be reverted before merging since this page is being worked on in different ticket